From 12128ce1aff92c3ca1ee37214f3fb9a3ac8f5d27 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Thu, 18 Jun 2015 10:54:26 +0800
Subject: [PATCH] MLK-11123-15 usb: otg: remove bcdOTG from usb_otg_descriptor

Remove bcdOTG from usb_otg_descriptor since we introduce usb_otg20_descriptor
for OTG 2.0 or above, and usb_otg_descriptor is only for OTG 1.x.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 include/uapi/linux/usb/ch9.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/uapi/linux/usb/ch9.h b/include/uapi/linux/usb/ch9.h
index 53d139f..76a7f03 100644
--- a/include/uapi/linux/usb/ch9.h
+++ b/include/uapi/linux/usb/ch9.h
@@ -674,12 +674,6 @@ struct usb_otg_descriptor {
 	__u8  bDescriptorType;
 
 	__u8  bmAttributes;	/* support for HNP, SRP, etc */
-	__le16 bcdOTG;		/* OTG and EH supplement release number
-				 * in binary-coded decimal (i.e. 2.0 is 0200H).
-				 * This field identifies the release of the
-				 * OTG and EH supplement with which the device
-				 * and its descriptors are compliant.
-				 */
 } __attribute__ ((packed));
 
 /* USB_DT_OTG (from OTG 2.0 supplement) */
-- 
1.8.0

